WEAPON/SetWeaponDamageModifier.md

```diff
@@ -12,3 +12,19 @@ void _SET_WEAPON_DAMAGE_MODIFIER(Hash weaponHash, float damageAmount);
 ## Parameters
 * **weaponHash**: 
 * **damageAmount**: 
+
+```
```

Comment:
Shouldn't the description be above the parameters and the description for what the parameter does, under the parameters field?

Reply:
Ammended